implementations/micrometer-registry-azuremonitor/build.gradle

```diff
@@ -0,0 +1,9 @@
+apply plugin: 'org.junit.platform.gradle.plugin'
+
+dependencies {
+    compile project(':micrometer-core')
+    compile 'com.microsoft.azure:applicationinsights-core:2.1.2-SNAPSHOT'
```

Comment:
@dhaval24 This needs to have a release version.

Reply:
@izeye yes ofcourse. Soon 2.1.2 version should be released (likely before 1.1.0-rc goes for Micrometer) and I will update this.